packages/gatsby/src/db/nodes-query.js

```diff
@@ -1,26 +1,5 @@
-const { getQueryFields, hasFieldResolvers } = require(`./common/query`)
-
 const lokiRunQuery = require(`./loki/nodes-query`)
-const siftRunQuery = require(`../redux/run-sift`)
-
-function chooseQueryEngine(args) {
-  const { backend } = require(`./nodes`)
-
-  const { queryArgs, gqlType } = args
-  const { filter, sort, group, distinct } = queryArgs
-  const fields = getQueryFields({ filter, sort, group, distinct })
-
-  // NOTE: `hasFieldResolvers` is also true for Date fields
-  if (
-    backend === `loki` &&
-    !args.nodes &&
-    !hasFieldResolvers(gqlType, fields)
```

Comment:
i think `hasFieldResolvers` is not used anymore and can be removed from `db/common/query`?